src/screens/dashboard/components/NotificationThumbnail.tsx

```diff
@@ -14,21 +14,24 @@ type NotificationThumbnailProps = {
 export const NotificationThumbnail = (props: NotificationThumbnailProps) => {
   const { user } = useUserContext()
 
+  const otherUserPhoto = user?.photo && NOTIFICATION_FROM_APP.includes(props.type)
+  const noPhoto = !props.author.photo && !user?.photo && NOTIFICATION_FROM_APP.includes(props.type)
+
   return (
     <>
-      {props.author.photo && (
+      {!!props.author.photo && (
         <FastImage
           source={{ uri: props.author.photo }}
           style={{ borderColor: props.author.userColor, width: 56, borderLeftWidth: 16 }}
         />
       )}
-      {user?.photo && NOTIFICATION_FROM_APP.includes(props.type) && (
+      {otherUserPhoto && (
```

Comment:
```suggestion
      {!!otherUserPhoto && (
```
an empty string is a falsy value and so this expression evaluates to empty string. And since it is not wrapped with the Text tag the app will crash. In reality the user photo should never be an empty string, but it's better to be covered than sorry